docs/request.md

```diff
@@ -272,27 +273,27 @@ Next, you need to create a [`FormData`](https://developer.mozilla.org/en/docs/We
 
 ```javascript
 function upload(e) {
-	var file = e.target.files[0]
+  var file = e.target.files[0]
 
-	var data = new FormData()
-	data.append("myfile", file)
+  var data = new FormData()
+  data.append("myfile", file)
```

Comment:
I think it would be better if this used `body` instead of `data` to make it clear what option should be used.

Reply:
Agreed.

Reply:
OK.